src/state/controls/Control.spec.ts

```diff
@@ -13,6 +13,11 @@ const buttonData = {
     text: 'foo',
     etag: '1234',
 };
+const buttonData2 = {
```

Comment:
Move cost into `buttonData` and just use that. Should be no need to create another object.